{
  "info": {
    "author": "b0shka",
    "author_email": "user.b0shka@gmail.com",
    "bugtrack_url": null,
    "classifiers": [
      "Intended Audience :: Developers",
      "License :: OSI Approved :: MIT License",
      "Operating System :: OS Independent",
      "Programming Language :: Python :: 3.8"
    ],
    "description": "# vk_face_recognition\nA program for creating a database with human biometrics based on photos. \nPhotos will be automatically downloaded from [VKontakte](https://vk.com). \nSubsequently, it will be possible to search for a person using the created database. \nThe program was created based on the `face_recognition` and `vk_api` libraries\n\n[![PyPI](https://img.shields.io/pypi/v/vk_face_recognition)](https://pypi.org/project/vk-face-recognition/)\n[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/vk-face-recognition)](https://pypi.org/project/vk-face-recognition/)\n\n____\n## Installation\n\n#### Requirements\n* Python 3.8+\n* Linux, Windows or macOS\n\n#### Installing on Linux or Mac\n```\npip3 install vk_face_recognition\n```\n\n#### Installing on Windows\n```\npip install vk_face_recognition\n```\n\n\n## Usage\n\n#### Initial setup\nTo get started, you need to add your token_vk received on the [site](https://vkhost.github.io/), to the file `main.py` to a string\n\n```\nself.session = vk_api.VkApi(token='your_token')\n```\n\n#### Using the Library\n```\nfrom vk_face_recognition import Vk_recognition\n\nvk_recognition = Vk_recognition()\nvk_recognition.start_recognition(age_min=25, age_max=26, gender=1, city='Moscow')\n```\n\nWhen running this code, a file will be created first `users_id.txt` with the user id according to the specified parameters of `age_min`, `age_max`, `gender` and `city`. Then the user's photos will be downloaded to the `data` folder. And then these photos will be used to recognize faces and create face parameters and write them to a separate file in the `pickle_files` folder\n\n#### Search by photo\nAfter creating the database you need, you can search through the photo in the following way\n\n```\nfrom vk_face_recognition import Vk_recognition\n\nsearch = Vk_recognition()\nsearch.find_people('path to the photo')\n```\n\nA search will be performed on the database you created and, if it matches, it will report this",
    "description_content_type": "text/markdown",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "https://github.com/b0shka/vk_face_recognition",
    "keywords": "",
    "license": "MIT license",
    "maintainer": "",
    "maintainer_email": "",
    "name": "vk-face-recognition",
    "package_url": "https://pypi.org/project/vk-face-recognition/",
    "platform": "",
    "project_url": "https://pypi.org/project/vk-face-recognition/",
    "project_urls": {
      "Homepage": "https://github.com/b0shka/vk_face_recognition"
    },
    "release_url": "https://pypi.org/project/vk-face-recognition/1.1.5/",
    "requires_dist": null,
    "requires_python": "",
    "summary": "Creating a database with biometrics of persons and subsequent search on it",
    "version": "1.1.5",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 10832665,
  "releases": {
    "1.1.3": [
      {
        "comment_text": "",
        "digests": {
          "blake2b_256": "b4d8661da14353261e5be37887f9ef39f9064da7c65d866f9cb27b74c0ac8402",
          "md5": "d9ec0bdc2604d5bc6e5780b00c8f99b4",
          "sha256": "28cd4e058e1efd3f6a3a11bd27fdb715447163bbf0c7e1cdd3807370be72a092"
        },
        "downloads": -1,
        "filename": "vk_face_recognition-1.1.3.tar.gz",
        "has_sig": false,
        "md5_digest": "d9ec0bdc2604d5bc6e5780b00c8f99b4",
        "packagetype": "sdist",
        "python_version": "source",
        "requires_python": null,
        "size": 4531,
        "upload_time": "2021-07-03T19:37:28",
        "upload_time_iso_8601": "2021-07-03T19:37:28.196591Z",
        "url": "https://files.pythonhosted.org/packages/b4/d8/661da14353261e5be37887f9ef39f9064da7c65d866f9cb27b74c0ac8402/vk_face_recognition-1.1.3.tar.gz",
        "yanked": false,
        "yanked_reason": null
      }
    ],
    "1.1.4": [
      {
        "comment_text": "",
        "digests": {
          "blake2b_256": "98f761dfbb1b9dc1ac6e05e5dddf0bddc99f47c87485b6d45fb5de3143833bd2",
          "md5": "d38a87add1485f5fc1b900f8ae29c2c9",
          "sha256": "f083d7205988ab5b3c8a2a4a73eddebf68f51456045cdd0f0fd4205fb2585f89"
        },
        "downloads": -1,
        "filename": "vk_face_recognition-1.1.4.tar.gz",
        "has_sig": false,
        "md5_digest": "d38a87add1485f5fc1b900f8ae29c2c9",
        "packagetype": "sdist",
        "python_version": "source",
        "requires_python": null,
        "size": 4566,
        "upload_time": "2021-07-03T20:01:45",
        "upload_time_iso_8601": "2021-07-03T20:01:45.274358Z",
        "url": "https://files.pythonhosted.org/packages/98/f7/61dfbb1b9dc1ac6e05e5dddf0bddc99f47c87485b6d45fb5de3143833bd2/vk_face_recognition-1.1.4.tar.gz",
        "yanked": false,
        "yanked_reason": null
      }
    ],
    "1.1.5": [
      {
        "comment_text": "",
        "digests": {
          "blake2b_256": "faccd149b2e162003df29336b014ca0de36815c3f56d864c280146dfaf410d95",
          "md5": "c36202fc8cd31356d0c0a560a92f6ae9",
          "sha256": "845996cf347e6364def74860ffb07f63b3676c8d19a8783190890da3ae3f6aee"
        },
        "downloads": -1,
        "filename": "vk_face_recognition-1.1.5.tar.gz",
        "has_sig": false,
        "md5_digest": "c36202fc8cd31356d0c0a560a92f6ae9",
        "packagetype": "sdist",
        "python_version": "source",
        "requires_python": null,
        "size": 4547,
        "upload_time": "2021-07-06T08:09:24",
        "upload_time_iso_8601": "2021-07-06T08:09:24.983931Z",
        "url": "https://files.pythonhosted.org/packages/fa/cc/d149b2e162003df29336b014ca0de36815c3f56d864c280146dfaf410d95/vk_face_recognition-1.1.5.tar.gz",
        "yanked": false,
        "yanked_reason": null
      }
    ]
  },
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "faccd149b2e162003df29336b014ca0de36815c3f56d864c280146dfaf410d95",
        "md5": "c36202fc8cd31356d0c0a560a92f6ae9",
        "sha256": "845996cf347e6364def74860ffb07f63b3676c8d19a8783190890da3ae3f6aee"
      },
      "downloads": -1,
      "filename": "vk_face_recognition-1.1.5.tar.gz",
      "has_sig": false,
      "md5_digest": "c36202fc8cd31356d0c0a560a92f6ae9",
      "packagetype": "sdist",
      "python_version": "source",
      "requires_python": null,
      "size": 4547,
      "upload_time": "2021-07-06T08:09:24",
      "upload_time_iso_8601": "2021-07-06T08:09:24.983931Z",
      "url": "https://files.pythonhosted.org/packages/fa/cc/d149b2e162003df29336b014ca0de36815c3f56d864c280146dfaf410d95/vk_face_recognition-1.1.5.tar.gz",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}